go/common/identity/identity.go

```diff
@@ -30,31 +32,124 @@ const (
 	tlsCertFilename = "tls_identity_cert.pem"
 )
 
+// ErrCertificateRotationForbidden is returned by RotateCertificates if
+// TLS certificate rotation is forbidden.  This happens when rotation is
```

Comment:
```suggestion
// TLS certificate rotation is forbidden. This happens when rotation is
```

Reply:
This is not a typo, I prefer using double-spaces at the end of sentences in monospace fonts, because it makes sentence boundaries clearer and easier to read.